app/lib/admin/tools/block_publisher_and_all_members.dart

```diff
@@ -37,19 +36,21 @@ Future<String> executeRemovePublisherAndBlockAllMembers(
 
   if (command == 'list') {
     return output.toString();
-  } else if (command == 'delete') {
+  } else if (command == 'block') {
     for (final m in members) {
       await accountBackend.updateBlockedFlag(m.userId, true);
     }
 
     final publisherKey = dbService.emptyKey.append(Publisher, id: publisherId);
     await withRetryTransaction(dbService, (tx) async {
-      tx.delete(publisherKey);
+      final p = await tx.lookupValue<Publisher>(publisherKey);
+      p.markForBlocked();
+      tx.insert(p);
       for (final m in members) {
         tx.delete(publisherKey.append(PublisherMember, id: m.userId));
```

Comment:
Is this a good idea, that makes it kind of non-reversible.

Reply:
Hm, I wanted to do it in a follow-up change, but since we already change the admin tool name, this could be also changed here.